test/integration_test_helper.rb

```diff
@@ -21,7 +21,15 @@
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
 
-javascript_driver = ENV.fetch("JS_TEST_DRIVER") { ENV['DEBUG_JS_TEST'] ? :selenium_chrome : :selenium_chrome_headless }.to_sym
+javascript_driver = ENV.fetch("JS_TEST_DRIVER") do
+  if ENV['SHOW_JS_LOG']
+    :logging_selenium_chrome
+  elsif ENV['DEBUG_JS_TEST']
+    :selenium_chrome
+  else
+    :selenium_chrome_headless
+  end
+end.to_sym
 
 def chrome_options
   options = Selenium::WebDriver::Chrome::Options.new
```

Comment:
Any thoughts on implementing it here?
```suggestion
  options = Selenium::WebDriver::Chrome::Options.new
  options.add_option("goog:loggingPrefs", {browser: 'ALL'})  if ENV['SHOW_JS_LOG']
```

I think that gives you the same result but with less code.

Reply:
Thanks, didnt realize it will work this simply, updated!